src/VkLayer_GF_amber_scoop/src/vulkan_commands.cc

```diff
@@ -16,6 +16,10 @@
 
 #include <VkLayer_GF_amber_scoop/draw_call_tracker.h>
 
+// <algorithm> is used by GCC's libstdc++, but not LLVM's libc++.
```

Comment:
The include above should not use < and > and I guess should come below the standard library includes.